handlers/root.go

```diff
@@ -5,9 +5,110 @@ import (
 	"net/http"
 
 	"github.com/swift-sunshine/swscore/log"
+	"github.com/swift-sunshine/swscore/pkg/kubernetes"
+	"github.com/swift-sunshine/swscore/pkg/prometheus"
+
 )
 
+// This method shows how to use the API to fetch and print all info that would be necessary
+// for listing services and details.
+// It is temporal, so don't be afraid to delete it when handlers are in place.
+func demoFetchAllServicesAndPrintThem(w http.ResponseWriter) {
+	istioClient, err := kubernetes.NewClient()
```

Comment:
This is a demo so it's fine, but in the final code I think clients should be initialized in parallel with goroutines.

Reply:
yes, it's demo purposes.
Also, perhaps it made sense to initialize the clients somewhere and reuse them.
I haven't tested but I wanted to provide a plain demo method to show the main features without enter in optimizations.